packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -177,6 +218,30 @@ export function BlockTypesTab(
 						);
 					}
 				) }
+
+				{ ( items.length === 0 ||
+					availableCategories.length === 1 ) && (
```

Comment:
Would be great to comment on this condition. Why show this when there's 1 category available and not 2?

Reply:
> Why show this when there's 1 category available and not 2?

Pure convention 🤷🏻